styles/variables-base.less

```diff
@@ -84,18 +84,6 @@
 @agate-datetimepicker-label-transform: inherit;
 @agate-datetimepicker-cell-transform: inherit;
 
-// Divider
-// ---------------------------------------
-@agate-divider-border-width: inherit;
-@agate-divider-font-size: inherit;
-@agate-divider-font-weight: inherit;
-@agate-divider-height: inherit;
-@agate-divider-spacing: @agate-component-spacing;
```

Comment:
Should we have `Heading` replacements for `@agate-divider-spacing` and `@agate-divider-font-ligatures`?

Reply:
`@agate-divider-spacing` is already present in Heading as `@agate-heading-spacing-medium`
We don't need that anymore `@agate-divider-font-ligatures`